integration/README.md

```diff
@@ -34,7 +34,7 @@ Some integration tests need a proxy. You can use [mitmproxy] or [squid].
 
 ### Start local server
 
-You can use the scripts [`bin/test/test_prerequites.sh`] / [`bin/test/test_prerequites.ps1`] depending on your OS to start the 
+You can use the scripts [`bin/test/test_prerequisites.sh`] / [`bin/test/test_prerequisites.ps1`] depending on your OS to start the 
```

Comment:
I would have renamed these files if they were actually spelled this way, but they weren't...